LeaderboardBackend/Models/Annotations/PlayersMaxAttribute.cs

```diff
@@ -0,0 +1,29 @@
+using LeaderboardBackend.Models.Requests.Categories;
+using System.ComponentModel.DataAnnotations;
+
+namespace LeaderboardBackend.Models.Annotations;
+
+public class PlayersMaxAttribute : ValidationAttribute
+{
+	public string GetErrorMessage(List<string> errors) =>
+		$"Your password has the following errors: {string.Join("; ", errors)}";
```

Comment:
Probably not a password anymore ;)

Reply:
Yeah I forgot to remove that method. It's not needed anymore.